src/_shared/js/capi-multiple.js

```diff
@@ -6,6 +6,8 @@ import { generatePicture } from './capi-images.js';
 import { setEditionLink } from './ads';
 import { URLSearchParams } from './utils';
 
+const clickMacro = '%%CLICK_URL_UNESC%%';
```

Comment:
Would it be worth adding this as a constant elsewhere where ''%%CLICK_URL_UNESC%%' is hardcoded?